### DISCUSSION TITLE
Exporting Songlist with Song Name and Artist

### DISCUSSION THREAD

**Discussion by SonmezYigithan** [sandiz/rs-manager#132] [Ideas]
Hi,

I'm trying to export my song lists and add them to Spotify. 
Right now when I export a setlist, it exports only dlc/cdlc file names. ( like "JoniBoth" for Joni Mitchell - Both Sides Now )
Maybe export names and artists can be an option when exporting.

**Comment by sandiz** [comment]
fixed with release [3.0.16](https://github.com/sandiz/rs-manager/releases/tag/v3.0.16)
